README.md

```diff
@@ -2,77 +2,80 @@
 
 # DDEV-CRON <!-- omit in toc -->
 
-- [Intro](#intro)
+- [Introduction](#introduction)
 - [Getting started](#getting-started)
 - [Implementation](#implementation)
 - [Examples](#examples)
+  - [TYPO3 scheduler](#typo3-scheduler)
+  - [Drupal cron](#drupal-cron)
+  - [Laravel cron](#laravel-cron)
 
-## Intro
+## Introduction
 
 This DDEV add-on helps to execute a command in the web container based on a cron schedule. Cron is a classic Linux/Unix service with a well-known configuration syntax.
 
-The add-on
+The add-on:
 
 - Installs and runs the cron service inside the web container
-- Adds a sample cron configuration that adds to a file every minute.
+- Adds an example job that writes out the current time.
 - Required DDEV v1.19.3 or higher.
 
-*This extension is designed to be a generic implentation. See [Running TYPO3 Cron inside the web container](https://github.com/ddev/ddev-contrib/tree/master/recipes/cronjob) for a specific example of a manual setup.*
+*This extension is designed to be a generic implementation. See [Running TYPO3 Cron inside the web container](https://github.com/ddev/ddev-contrib/tree/master/recipes/cronjob) for a specific example of a manual setup.*
 
 ## Getting started
 
-- Install the add-on with `ddev get ddev/ddev-cron`
-- Update the provided `.ddev/config.cron.yaml` as you see fit with your expected cron jobs (and remove the demonstration line). You can also just add those demonstration lines to your `.ddev/config.yaml` and delete the `.ddev/config.cron.yaml`.
-- `ddev restart`
+- Install the DDEV cron add-on:
 
-## Implementation
+  ```shell
+  ddev get ddev/ddev-browsersync
+  ```
 
-The provided `web-build/Dockerfile.ddev-cron` and `web-build/cron.conf` configure the traditional cron daemon to run inside the web container.
+- Update `./.ddev/web-build/custom.cron` with your required commands.
+  - Remove `#ddev-generated` to prevent DDEV from overriding the file.
+- Custom the cron service, if required, by updating `./.ddev/web-build/cron.conf`
+  - Remove `#ddev-generated` to prevent DDEV from overriding the file.
```

Comment:
Again, if we can use override files instead of editing the installed files it will be better. We want people to always be able to `ddev get ddev/ddev-cron` without having trouble with their changes.

Reply:
I wonder if it's OK to keep the `cron.conf` file as `#ddev-generated` as this is what we do with nginx, postgres. The current defaults seem sensible for different projects.